src/components/TodoNavbar.tsx

```diff
@@ -0,0 +1,25 @@
+export default function TodoNavbar() {
```

Comment:
Why this component is declared as function ?

Reply:
Done